### PR TITLE
Rename se.sjoerd.DatMan to se.sjoerd.Graphs

### PR DESCRIPTION
**Pull request by SjoerdB93**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission
-------------------------------------

Early in the development of Graphs, the project was called DatMan (short for Data Manipulator). Along the way, I've made more attempts to more closely follow the GNOME HIG guidelines, which included a name-change from DatMan to Graphs. Now the project is on rails and doing well enough, we feel it's time to also change the app-id to follow the same conventions. 

The current, very different, app-id is confusing both for users and developers. Especially when working from the cli. So a name-change of the app-id feels needed, even if it's inconvenient. The name itself is set in stone now, so a new name change will not occur, apologies for the inconvenience.

### CODE REVIEW

**Comment by SjoerdB93** [comment]
bot, build se.sjoerd.Graphs

**Comment by SjoerdB93** [comment]
bot, build se.sjoerd.Graphs

**Comment by SjoerdB93** [comment]
bot, build se.sjoerd.Graphs

**Comment by cmkohnen** [comment]
Maybe to add: the app in question to be replaced is [this one](https://flathub.org/apps/details/se.sjoerd.DatMan), which would with the acceptance of this pr be deprecated / asked to be removed

**Comment by SjoerdB93** [comment]
bot, build se.sjoerd.Graphs

**Comment by SjoerdB93** [comment]
bot, build se.sjoerd.Graphs

**Comment by SjoerdB93** [comment]
bot, build se.sjoerd.Graphs

**Comment by SjoerdB93** [comment]
There was an issue with the desktop file. Before submitting the application here for renaming, I tested it with the .flatpakref built by Gnome Builder and that worked fine. After installing using the link that flathubbot send me, it also worked fine from the CLI using `flatpak run se.sjoerd.Graphs`, but it refused to launch using the GUI.

I have now found the culprit. The exec command in the .desktop file was still trying to launch `datman`. This does not get picked up by the flatpakref which generates the desktop file differently. This should all be good to go with the latest build that I just triggered.

**Comment by SjoerdB93** [comment]
Latest build indeed works fine, this is ready for review :)

Apologies for the many builds after submission, but this bug didn't get picked up in the regular testing that I typically do with the installed flatpakref I get from GNOME builder.

File: se.sjoerd.Graphs.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
```
Comment (TingPing) [inline se.sjoerd.Graphs.json:23]: This entire list seems unused? You can remove it and look in `files` to see whats actually extra, if anything.

Comment (SjoerdB93) [reply se.sjoerd.Graphs.json:23]: This list was part of the manifest template in GNOME Builder, but I'll take a look if any of this is necessary.

Comment (TingPing) [reply se.sjoerd.Graphs.json:23]: This is a minor detail so you can look at it later.

**Pull request closed without merging**